media/js/devreg/lookup-tool.js

```diff
@@ -51,6 +51,75 @@ require(['prefetchManifest']);
         $(this).next('button').removeClass('disabled');
     });
 
+    $('.add-group button').click(_pd(function() {
+        var button = $(this);
+        var select = button.parent().parent().find('select');
```

Comment:
Also, you can directly set `select` to be the actual DOM object instead of a jQuery object and avoid a bunch of `[0]` below.